code/modules/clothing/masks/gasmask.dm

```diff
@@ -83,6 +83,7 @@
 	resistance_flags = FLAMMABLE
 	actions_types = list(/datum/action/item_action/adjust)
 	dog_fashion = /datum/dog_fashion/head/clown
+	species_exception = list(/datum/species/golem/bananium)
```

Comment:
you forgot to also let bananium golems wear sexy cown wigs and masks

Reply:
They're a child of this mask pretty sure. I'll double check though.

Reply:
they aren't, I checked myself